SUMMARY.md

```diff
@@ -9,6 +9,7 @@
     * [Life-cycle](vm-life-cycle.md)
     * [Access](access.md)
     * [Ephemeral Registry Disks](registry-disk.md)
+    * [Startup Scripts](startup-scripts.md)
```

Comment:
I am not sure what would be a better name, but at least I don't think about initial provisioning and cloud-init when I read `Startup Scripts`. I associate that term more with init systems like systemd or SysV.

Reply:
https://cloud.google.com/compute/docs/startupscript GCE uses this term at least.  I don't have a better name right now.  I was trying to leave out any mention of specific tech (like cloud-init) in the title since it should more generic than that in the future.

Reply:
ok sounds good to me then.